planner_cspace/src/planner_3d.cpp

```diff
@@ -760,29 +752,63 @@ class Planner3dNode
           geometry_msgs::Point32 point;
           point.x = x;
           point.y = y;
-          switch (debug_out_)
-          {
-            case DEBUG_HYSTERESIS:
-              if (cost_estim_cache_[p] == FLT_MAX)
-                continue;
+          if (cost_estim_cache_[p] == FLT_MAX)
+            continue;
+          point.z = cost_estim_cache_[p] / 500;
```

Comment:
where does `500` comes from?

Reply:
It's a magic parameter of the visualization used in the master branch. In this PR, it just keep compatibility with the previous behaviour.